### PR TITLE
Augmentation tutorials

### PR DESCRIPTION
**Pull request by BorisJov**
Completes issue #1473

### CODE REVIEW

File: Assets/MirageXR/Managers/Activity/ActivityManager.cs [hunk 1]
```diff
@@ -284,7 +285,7 @@ private async Task ActivateAction(string id)
                     }
 
                     await ActivateAction(action);
-                    TutorialManager.Instance.InvokeEvent(TutorialManager.TutorialEvent.ACTION_STEP_ACTIVATED);
```
Comment (fwild) [inline Assets/MirageXR/Managers/Activity/ActivityManager.cs]: There is a direct lib-lee event for that: LearningExperienceEngine.EventManager.OnStepActivated += ACTION_STEP_ACTIVATED

File: Assets/MirageXR/Managers/Activity/ActivityManager.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿using i5.Toolkit.Core.VerboseLogging;
+using NSubstitute.Core;
```
Comment (fwild) [inline Assets/MirageXR/Managers/Activity/ActivityManager.cs]: Is this needed?

**Review by fwild — CHANGES_REQUESTED** [review]

File: Assets/MirageXR/Managers/Activity/ActivityManager.cs (hunk 1 above)
Comment (BorisJov) [reply Assets/MirageXR/Managers/Activity/ActivityManager.cs]: Artefact of an earlier version. It's no longer present

File: Assets/MirageXR/Managers/Activity/ActivityManager.cs (hunk 2 above)
Comment (BorisJov) [reply Assets/MirageXR/Managers/Activity/ActivityManager.cs]: Artefact of an earlier version. It's no longer present

File: Assets/MirageXR/Managers/Activity/ActivityManager.cs
Comment (cawke) [inline Assets/MirageXR/Managers/Activity/ActivityManager.cs:1]: I think the empty file can be deleted

**Review by cawke — CHANGES_REQUESTED** [review]
small note

File: Assets/MirageXR/Managers/Activity/ActivityManager.cs
Comment (BorisJov) [reply Assets/MirageXR/Managers/Activity/ActivityManager.cs:1]: @fwild Did you leave the Activity Manager as an empty file intentionally?

Comment (fwild) [reply Assets/MirageXR/Managers/Activity/ActivityManager.cs:1]: This file does not exist anymore - this is now in Packages/com.wekit-ecs.lib-lee/Runtime/DataManipulation/ActivityManager.cs. So: yes, this one can be removed!

Comment (BorisJov) [reply Assets/MirageXR/Managers/Activity/ActivityManager.cs:1]: It's been removed now.

**Review by fwild — APPROVED** [review]

**Review by cawke — APPROVED** [review]

**Pull request merged**